<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="action_N_Ctrl_N"/>
    <addaction name="action_O_Ctrl_O"/>
    <addaction name="action_S_Ctrl_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="action_U_Ctrl_Z"/>
    <addaction name="action_Ctrl_Shift_Z"/>
    <addaction name="action_T_Ctrl_X"/>
    <addaction name="action_C_Ctrl_C"/>
    <addaction name="action_V"/>
    <addaction name="separator"/>
    <addaction name="action_R"/>
    <addaction name="action_F"/>
    <addaction name="separator"/>
    <addaction name="action_A_2"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(O)</string>
    </property>
    <addaction name="action_W"/>
    <addaction name="action_L"/>
    <addaction name="action_F_2"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(V)</string>
    </property>
    <addaction name="action_T"/>
    <addaction name="action_S"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="action_A_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_N_Ctrl_N"/>
   <addaction name="action_O_Ctrl_O"/>
   <addaction name="action_S_Ctrl_S"/>
   <addaction name="action_A"/>
   <addaction name="separator"/>
   <addaction name="action_U_Ctrl_Z"/>
   <addaction name="action_Ctrl_Shift_Z"/>
   <addaction name="action_T_Ctrl_X"/>
   <addaction name="action_C_Ctrl_C"/>
   <addaction name="action_V"/>
   <addaction name="separator"/>
   <addaction name="action_F"/>
   <addaction name="action_R"/>
   <addaction name="separator"/>
   <addaction name="action_W"/>
   <addaction name="action_L"/>
   <addaction name="action_F_2"/>
   <addaction name="separator"/>
   <addaction name="action_T"/>
   <addaction name="action_S"/>
  </widget>
  <action name="action_N_Ctrl_N">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\new.png</normaloff>D:\QtFile\TextEditor\resources\new.png</iconset>
   </property>
   <property name="text">
    <string>新建(N)</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_O_Ctrl_O">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\open.png</normaloff>D:\QtFile\TextEditor\resources\open.png</iconset>
   </property>
   <property name="text">
    <string>打开(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_S_Ctrl_S">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\save.png</normaloff>D:\QtFile\TextEditor\resources\save.png</iconset>
   </property>
   <property name="text">
    <string>保存(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_A">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\another save.png</normaloff>D:\QtFile\TextEditor\resources\another save.png</iconset>
   </property>
   <property name="text">
    <string>另存为(A)</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_U_Ctrl_Z">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\chexiao.png</normaloff>D:\QtFile\TextEditor\resources\chexiao.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_Ctrl_Shift_Z">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\恢复.png</normaloff>D:\QtFile\TextEditor\resources\恢复.png</iconset>
   </property>
   <property name="text">
    <string>恢复()</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_T_Ctrl_X">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\剪切.png</normaloff>D:\QtFile\TextEditor\resources\剪切.png</iconset>
   </property>
   <property name="text">
    <string>剪切(T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_C_Ctrl_C">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\复制.png</normaloff>D:\QtFile\TextEditor\resources\复制.png</iconset>
   </property>
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_V">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\粘贴.png</normaloff>D:\QtFile\TextEditor\resources\粘贴.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_F">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\查询.png</normaloff>D:\QtFile\TextEditor\resources\查询.png</iconset>
   </property>
   <property name="text">
    <string>查找(F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_R">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\替换.png</normaloff>D:\QtFile\TextEditor\resources\替换.png</iconset>
   </property>
   <property name="text">
    <string>替换(R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>退出(X)</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>全选(A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_W">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\自动换行.png</normaloff>D:\QtFile\TextEditor\resources\自动换行.png</iconset>
   </property>
   <property name="text">
    <string>自动换行(W)</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_L">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\显示行号.png</normaloff>D:\QtFile\TextEditor\resources\显示行号.png</iconset>
   </property>
   <property name="text">
    <string>显示行号(L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_F_2">
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\字体.png</normaloff>D:\QtFile\TextEditor\resources\字体.png</iconset>
   </property>
   <property name="text">
    <string>字体(F)</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>字体颜色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>字体背景色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>编辑器背景色</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_T">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\工具栏.png</normaloff>D:\QtFile\TextEditor\resources\工具栏.png</iconset>
   </property>
   <property name="text">
    <string>工具栏(T)</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_S">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>D:\QtFile\TextEditor\resources\状态栏.png</normaloff>D:\QtFile\TextEditor\resources\状态栏.png</iconset>
   </property>
   <property name="text">
    <string>状态栏(S)</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_A_3">
   <property name="text">
    <string>关于(A)</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
